erdpy/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes will be documented in this file.
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## [1.0.3] - 28.12.2020
```

Comment:
Still do not know why do we have to keep a change log if we have git. Git is an excellent tool for keeping track of the changes. @andreibancioiu

Reply:
Currently, since elrond-sdk is a mono-repo, it isn't very easy to list important changes, fixes and new features in GitHub releases (though we can do so, but then, the community has to scan & filter them). Therefore, we keep a separate changelog for each component (erdpy, erdjs, erdgo etc.), on this purpose. Generally, changelogs are needed / desired for some "release platforms" such as the VsCode Marketplace.

https://keepachangelog.com/en/1.0.0/